zip-0032.rst

```diff
@@ -373,6 +441,27 @@ Wallets implementing Sprout ZIP 32 derivation MUST support the following path:
 
 * :math:`m_\mathsf{Sprout} / purpose' / coin\_type' / account' / address\_index`.
 
+Orchard key path
+----------------
+
+Orchard supports diversified addresses with the same spending authority (like Sapling). A group of such
+addresses shares the same full viewing key and incoming viewing key, and so creating as many unlinkable
+addresses as needed does not increase the cost of scanning the block chain for relevant transactions.
+
+The above key path levels include an account identifier, which in all user interfaces is represented as a
+"bucket of funds" under the control of a single spending authority. Therefore, wallets implementing Orchard
+ZIP 32 derivation MUST support the following path for any account in range :math:`\{ 0\,.\!. 2^{31} - 1 \}`:
+
+* :math:`m_\mathsf{Orchard} / purpose' / coin\_type' / account'`.
+
+Furthermore, wallets MUST support generating the default payment address (corresponding to the default
+diversifier for Orchard) for any account they support. They MAY also support generating a stream of payment
+addresses for a given account, if they wish to maintain the user experience of giving a unique address to
+each recipient.
```

Comment:
The motivation for this is primarily security, not maintaining user experience. See the note at the end of https://zips.z.cash/protocol/protocol.pdf#addressesandkeys .